File: source/repos/BlackJack/BlackJack.ViewModels/GameViews/PlayGameView.cs
using BlackJack.DataAccess.Enums;
using System.Collections.Generic;

namespace BlackJack.ViewModels.GameViews
{
    public class PlayGameView
    {
        public int NumberOfBots { get; set; }
        public StatusType Status { get; set; }
        public string Winner { get; set; }

        public PlayerPlayGameView Player { get; set; }
        public List<BotPlayGameViewItem> Bots { get; set; }

        public PlayGameView()
        {
            Player = new PlayerPlayGameView();
            Bots = new List<BotPlayGameViewItem>();
        }
    }
    public class PlayerPlayGameView
    {
        public string Name { get; set; }
        public List<CardPlayGameViewItem> Cards { get; set; }
    }
    public class BotPlayGameViewItem
    {
        public string Name { get; set; }
        public List<CardPlayGameViewItem> Cards { get; set; }
    }
    public class CardPlayGameViewItem
    {
        public CardRankType Rank { get; set; }
        public CardSuitType Suit { get; set; }
    }
}

File: source/repos/BlackJack/BlackJack.WEB/ClientApp/src/app/shared/entities/game/play-game.view.ts
import { CardRankType } from 'src/app/shared/enums/card-rank-type.view';
import { CardSuitType } from 'src/app/shared/enums/card-suit-type.view';
import { StatusType } from 'src/app/shared/enums/status-type.enum.view';

export class PlayGameView {
    numberOfBots: number;
    status: StatusType;
    winner: string;
    player:PlayerPlayGameView;
    bots:BotPlayGameViewItem[] = [];
}
export class PlayerPlayGameView{
    name:string;
    cards:CardPlayGameViewItem[] = [];
}
export class BotPlayGameViewItem{
    name:string;
    cards:CardPlayGameViewItem[] = [];
}
export class CardPlayGameViewItem{
    rank:CardRankType;
    suit:CardSuitType;
}